src/sidebar.js

```diff
@@ -134,7 +134,6 @@ module.exports = {
             'kratos/quickstart',
             'kratos/install',
             'kratos/configuring',
-            'kratos/contributing'
```

Comment:
I think you should add redirects for the pages that get removed in the https://github.com/ory/docs/blob/master/vercel.json file. That's what I was asked to do when I removed/moved docs around ;)